### PR TITLE
Fix issue #168: intervals greater than one day were incorrect.

### PR DESCRIPTION
**Pull request by ftobia**
This is a proposed fix for issue #168

### CODE REVIEW

File: maya/core.py [hunk 1]
```diff
@@ -806,4 +806,6 @@ def intervals(start, end, interval):
     while current_timestamp.epoch < end.epoch:
         yield current_timestamp
 
-        current_timestamp = current_timestamp.add(seconds=interval.seconds)
+        current_timestamp = current_timestamp.add(
+            seconds=interval.total_seconds()
```
Comment (ftobia) [inline maya/core.py]: I think this is the right solution. 
I don't have a strong grasp on how microseconds interact with maya internals, so it's possible that a better solution here would be calculating number of seconds manually: `(td.seconds + td.days * 24 * 3600)` per the Python 2 docs. From reading the docs (linked below) it looks like accounting for microseconds causes us to lose accuracy over a timedelta of 270 years or so.

Python 2 docs: https://docs.python.org/2/library/datetime.html#datetime.timedelta.total_seconds
Python 3 docs: https://docs.python.org/3/library/datetime.html#datetime.timedelta.total_seconds

File: tests/test_maya_interval.py [hunk 2]
```diff
@@ -570,3 +570,16 @@ def test_interval_from_iso8601_duration():
 
     assert interval.start == s
     assert interval.end == e
+
+
+def test_issue_168_regression():
```
Comment (ftobia) [inline tests/test_maya_interval.py]: I'm all ears on how to improve this test. It looks like there aren't any explicit tests for the `maya.intervals(...)` functionality. I'm willing to try adding some if a maintainer will let me know what they'd like to see.

Comment (timofurrer) [reply tests/test_maya_interval.py]: IMHO: this test looks good! At least for intervals in a resolution of days. Maybe it makes sense to add tests for other intervals, like seconds, minutes and hours .. If you'd like some of those - I'm always happy to review!

File: maya/core.py (hunk 1 above)
Comment (timofurrer) [reply maya/core.py]: Looks good to me! :tada:

**Comment by ftobia** [comment]
@timofurrer ready for your review.

**Comment by timofurrer** [comment]
Awesome, thank you :tada:

**Pull request merged**